src/main/resources/jupyter/init-actions.sh

```diff
@@ -112,7 +111,7 @@ if [[ "${ROLE}" == 'Master' ]]; then
       gsutil cp ${JUPYTER_EXTENSION_URI} /etc
       JUPYTER_EXTENSION_ARCHIVE=`basename ${JUPYTER_EXTENSION_URI}`
       docker cp /etc/${JUPYTER_EXTENSION_ARCHIVE} ${JUPYTER_SERVER_NAME}:${JUPYTER_HOME}/${JUPYTER_EXTENSION_ARCHIVE}
-      docker exec -d ${JUPYTER_SERVER_NAME} ${JUPYTER_HOME}/install-jupyter-extension.sh ${JUPYTER_HOME}/${JUPYTER_EXTENSION_ARCHIVE}
+      docker exec -d ${JUPYTER_SERVER_NAME} ${JUPYTER_HOME}/jupyter_install_notebook_extension.sh ${JUPYTER_HOME}/${JUPYTER_EXTENSION_ARCHIVE}
```

Comment:
This also fixes https://github.com/DataBiosphere/leonardo/issues/221, but I'll leave the test case to @aweng98